### PR TITLE
Adding explicit description on when to sign which CLA

### PR DESCRIPTION
**Pull request by teone**
**Describe the PR**
Add clear wording on when to sign which CLA.

Preview:
Screen Shot 2020-08-13 at 11 32 15 AM

### CODE REVIEW

File: client/src/js/agreement/CreateAgreementContainer.jsx [hunk 1]
```diff
@@ -28,9 +25,45 @@ function CreateAgreementContainer () {
...
+          <Card>
+            <CardContent>
+              <Typography variant="h5" component="h2" className={classes.textCenter}>
+                When to sign an Individual CLA?
+              </Typography>
+              <Typography variant="body2" component="div">
+                I should sign this CLA if:
+                <ul>
+                  <li>I'm making contributions
+                    on my own free time and no-one is contracting me for them.</li>
```
Comment (bocon13) [inline client/src/js/agreement/CreateAgreementContainer.jsx:42]: I'm making contributions in my free time with my own resources. I am not being paid to contribute.

File: client/src/js/agreement/CreateAgreementContainer.jsx [hunk 2]
```diff
@@ -28,9 +25,45 @@ function CreateAgreementContainer () {
...
+        <Grid item xs={12} md={6}>
+          <Card>
+            <CardContent>
+              <Typography variant="h5" component="h2" className={classes.textCenter}>
+                When to sign an Institutional CLA?
+              </Typography>
+              <Typography variant="body2" component="div">
+                I should sign this CLA if:
+                <ul>
+                  <li>I'm making contributions on behalf of my employer</li>
```
Comment (bocon13) [inline client/src/js/agreement/CreateAgreementContainer.jsx:57]: Someone is paying me to contribute to the project
I'm making contributions on behalf of my employer

In these circumstances, it is very likely that the person paying you to do the work owns the rights to it. They need to sign the CLA for these contributions.

Comment (teone) [reply client/src/js/agreement/CreateAgreementContainer.jsx:57]: Is this what you were suggesting?
```
-                  <li>I'm making contributions on behalf of my employer</li>
-                  <li>I'm emplying (or contracting with) someone to contribute to the project</li>
+                  <li>Someone is paying me to contribute to the project</li>
+                  <li>I'm employing (or contracting with) someone to contribute to the project</li>
```

Do we need to add `In these circumstances, it is very likely that the person paying you to do the work owns the rights to it. They need to sign the CLA for these contributions.`?

The next page question (`Are you authorized to sign legal agreements for your company?` should cover that part

**Comment by aindermitte** [comment]
"emplying" --> "employing"

File: client/src/js/agreement/CreateAgreementContainer.jsx (hunk 1 above)
Comment (zdw) [reply client/src/js/agreement/CreateAgreementContainer.jsx:42]: I'm making contributions of code I've created in my free time (not under contract), and I hold the copyright on my contributions.

File: client/src/js/agreement/CreateAgreementContainer.jsx (hunk 2 above)
Comment (zdw) [reply client/src/js/agreement/CreateAgreementContainer.jsx]: "I'm making contributions as an employee or under a contract or other agreement with a company, and that company holds the copyright and patents rights on my contributions."

**Pull request merged**